cmd/bootstrap/cmd/finalize.go

```diff
@@ -92,6 +93,8 @@ func addFinalizeCmdFlags() {
 	finalizeCmd.Flags().Uint64Var(&flagNumViewsInEpoch, "epoch-length", 4000, "length of each epoch measured in views")
 	finalizeCmd.Flags().Uint64Var(&flagNumViewsInStakingAuction, "epoch-staking-phase-length", 100, "length of the epoch staking phase measured in views")
 	finalizeCmd.Flags().Uint64Var(&flagNumViewsInDKGPhase, "epoch-dkg-phase-length", 1000, "length of each DKG phase measured in views")
+	finalizeCmd.Flags().BytesHexVar(&flagBootstrapRandomSeed, "random-seed", GenerateRandomSeed(), "The seed used to for DKG, Clustering and Cluster QC generation")
```

Comment:
I'm guessing this showed up as an addition because of this branch being updated with a more recent version of `master` than the dependent PR (https://github.com/onflow/flow-go/issues/806). Now that that PR is merged and this PR is targeting the feature PR, it doesn't appear as an addition, as expected.